docs/drilldowns/explore-underlying-data.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[explore-underlying-data]]
+== Explore data underlying visualizations
+
+Dashboard panels have "Explore underlying data" actions that will redirect
```

Comment:
Suggest combing the first two paragraphs:

> Dashboard panels have an **Explore underlying data** action that navigates you to Discover,
> where you can narrow your documents to the ones you use most often.
> This action is available for visualizations backed by a single index pattern.

Reply:
I've applied your suggestion, but I also used `*` around "Discover", like `*Discover*` as I've Discover been referenced like that in other places.

Also, instead of "you use most often", I've changed it to "likely used in the visualization", because those documents are not often used ones but the ones that were used to construct the visualization, plus, potentially some extra documents.